test/constructors.jl

```diff
@@ -18,8 +18,8 @@ module TestConstructors
 
     @test isequal(dt, DataTable(Any[NullableCategoricalVector(zeros(3)),
                                     NullableCategoricalVector(ones(3))]))
-    @test isequal(dt, DataTable(x1 = [0.0, 0.0, 0.0],
-                                x2 = [1.0, 1.0, 1.0]))
+    @test !isequal(dt, DataTable(x1 = [0.0, 0.0, 0.0],
```

Comment:
Not much point in testing that `dt` isn't equal to this. Rather add a test that the standard constructor produces `Vector` columns somewhere else (if that's not yet checked).